gatsby-ssr.js

```diff
@@ -31,3 +32,19 @@ export const wrapRootElement = ({ element }) => (
 )
 
 export const wrapPageElement = ({ element }) => <Layout>{element}</Layout>
+
+export const onRenderBody = ({ setHeadComponents }) => {
+  if (storeConfig.analytics.gtmContainerId) {
+    setHeadComponents([
+      <GoogleTagManager
+        key="gtm"
+        containerId={storeConfig.analytics.gtmContainerId}
+        enablePartytown
+      />,
+      <Partytown key="party" />,
+    ])
+  } else if (process.env.NODE_ENV === 'development') {
+    // eslint-disable-next-line
+    console.log('Add the gtmContainerId on storeConfig file')
```

Comment:
I think this would be better off as a warning. Something like: "`gtmContainerId` is not set. Google Tag Manager will not work until you provide a valid container id.".